packages/client/src/components/ProductListDropDown/DropDownView/DropDownView.style.css

```diff
@@ -14,7 +14,7 @@
 }
 
 .all-filters {
-  background: url('../../../assets/all-flter-icon.svg') no-repeat;
+  background: url('all-filter-icon.svg') no-repeat;
```

Comment:
I looked into this and this has changed since the last FP I mentored, the way you did it at first is indeed the correct way so sorry for the noise, please revert this line back to:
`background: url('../../../assets/all-filter-icon.svg') no-repeat;`